style3.css

```diff
@@ -116,13 +97,54 @@ ul li:hover #a{
 
 
 
+/* Set up the footer */
 /* Set up the footer */
 .footer{
 	background-color: #0C59CF;
+	grid-column-start: 1;
 	grid-column-end: span 4;
-	padding-top:25px;
 	text-align: center;
-	color:black;
 	height:50px;
 	color:white;
+	padding-top:20px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+
+
+user1{
+	color:#0C59CF;
+	margin-left:580px;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
+}
+
+password1{
+	color:#0C59CF;
+	margin-left:40px;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
+}
+
+section1, section2{
```

Comment:
27/09/2021: I create the the the section1 and section two which is holding the setting for the data for user and password column. Because it was a column so I setup a border. Also I setting the font style  and position for them so it should display underneath the heading that they belong for.